Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/demons.yml

```
@@ -0,0 +1,81 @@
+    - SimpleHostile
+  - type: Physics
+    bodyType: KinematicController
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.0
+        density: 0
+        mask:
+          - FlyingMobMask
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)